File: week-2/letter-l.c
#include <stdio.h>

void secret_shape(int n)
{
   for(int i = 1; i <= n; i++)
   {
     for(int j = 1; j <= n; j++)
     {
       if(j == 1 || i == n) printf("*");
     }
     printf("\n");
   }
   printf("\n");
}

int main()
{
   int start = 1;
   int end = 5;
   for(int i = 1; i <= 5; i++)
   {
      secret_shape(i);
   }
   return 0;
}

File: week-2/letter-l.py
def secret_shape(n):
    for i in range(1,n+1):
        for j in range(1,n+1):
            if (j == 1 or i == n):
                print("*",end="")
            else:
                print(" ",end="")
        print()

for i in range(1,6):
    secret_shape(i)
    print()
